src/client.js

```diff
@@ -139,7 +138,7 @@ const update = () => {
         state.saveSlots = savedState.saveSlots;
         state.lastResized = Date.now();
         state.context = context;
-        canvas.onmousedown =  onMouseDown;
+        canvas.onmousedown = onMouseDown;
```

Comment:
You have doubled up these event handlers now. For each of these we should do `canvas.on.... =` or `canvas.addEventListener(...)` but not both.